packages/react-components/react-persona/src/stories/Persona/PersonaAvatarSize.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { Persona } from '@fluentui/react-persona';
+import { AvatarSizes, makeStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(4, max-content)',
+    rowGap: '10px',
+  },
+});
+
+const sizes: AvatarSizes[] = [16, 20, 24, 28, 32, 36, 40, 48, 56, 64, 72, 96, 120, 128];
+
+export const AvatarSize = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      {sizes.map(size => (
```

Comment:
To keep the example code simpler, it's probably better to write out the example for each size (maybe just one example per size, with primary and secondary text). See AvatarSize.stories.tsx or PresenceBadgeSizes.stories.tsx.

Reply:
These size stories could also serve the purpose of showing how fixed sizing works. (Instead of separate stories for fixed sizing)